src/ParticularTemplates/NServiceBusEndpoint/Program.cs

```diff
@@ -26,8 +25,7 @@
 
 var builder = Host.CreateApplicationBuilder(args);
 #if (hosting == "WindowsService")
-builder.UseWindowsService();
-builder.Logging.AddEventLog();
+builder.Services.AddWindowsService();
```

Comment:
Does the AddWindowsService() include event log logging internally?

Reply:
Yep!